notes/stories/film/film-scifi-2020s.st.txt

```diff
@@ -197,6 +197,53 @@ powered exoskeleton [Dalton hooked up to his spine a robotic apparatus with long
 programmable person [Ray could be programmed to do Rising Spirit Tech corporation's bidding.]
 
 
+movie: Red Dwarf:The Promised Land (2020)
+=========================================
+
+:: Title
+Red Dwarf:The Promised Land
+
+:: Description
+Red Dwarf: The Promised Land is a 2020 British science-fiction comedy
+television film and the thirteenth installment of the British science-fiction
+sitcom, Red Dwarf.
+
+Synopsis: The Red Dwarf crew of Dave Lister, Arnold Rimmer, the Cat, and
+Kryten face-off against Rodon, the leader of a band of feral cats, who has
+pledged to annihilate all those who don't worship him as their god.
+
+:: Date
+2020-04-09
+
+:: References
+https://en.wikipedia.org/wiki/Red_Dwarf:_The_Promised_Land
+
+:: Ratings
+3 <paul>
```

Comment:
Typo?

Reply:
You'd think so, wouldn't you? But go figure it actually had a coherent plot.